app/browser/notifications/activityManager.js

```diff
@@ -54,6 +56,62 @@ class ActivityManager {
 function setEventHandlers(self) {
   self.ipcRenderer.on("enable-wakelock", () => wakeLock.enable());
   self.ipcRenderer.on("disable-wakelock", () => wakeLock.disable());
+
+  self.ipcRenderer.on('incoming-call-action', (event, action) => {
+    console.debug("ActionHTML", document.body.innerHTML);
+    const actionWrapper = document.querySelector('[data-testid="calling-actions"],[data-testid="msn-actions"]');
+    if (actionWrapper) {
+      const buttons = actionWrapper.querySelectorAll("button");
+      if (buttons.length > 0) {
+        switch (action) {
+          case 'ACCEPT_AUDIO':
+            if (buttons.length == 3) {
+              buttons[1].click();
+            }
+
```

Comment:
If you don't add a 'break' in here, it will execute the default section after doing the accept audio. Basically it will try to click the 1st button and then the last button (that can trigger both events and cause some strange issues.